pkg/cmd/fixtures.go

```diff
@@ -73,7 +75,7 @@ func (fc *FixturesCmd) runFixturesCmd(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	_, err = fixture.Execute(cmd.Context())
+	_, err = fixture.Execute(cmd.Context(), fc.apiVersion)
```

Comment:
could also pass `api-version` flag to `NewFixtureFromFile` like how the other flags are, but it made less sense as version gets set/changed within execute. if it would be better off in the other function though, lmk!

Reply:
i agree it is better to be passed into the execution method directly